### PR TITLE
Tweak dashboard styles.

### PR DESCRIPTION
**Pull request by mltucker**
I didn't run these yet so be sure to test this before merging :). I'll set up docker when I get a chance.

### CODE REVIEW

File: resources/js/Pages/Dashboard.vue [hunk 1]
```diff
@@ -218,15 +218,15 @@ if (isPushNotificationSupported()) {
                             </div>
                             <Link v-if="lesson.pivot.participation == 'teacher'"
                                 :href="route('lessons.edit', [lesson.id])">
-                            <SecondaryButton>Edit Lesson</SecondaryButton>
+                            <SecondaryButton className="mr-2">Edit Lesson</SecondaryButton>
```
Comment (Marco-Peter) [inline resources/js/Pages/Dashboard.vue:221]:
```suggestion
                            <SecondaryButton class="mr-2">Edit Lesson</SecondaryButton>
```

File: resources/js/Pages/Dashboard.vue [hunk 2]
```diff
@@ -218,15 +218,15 @@ if (isPushNotificationSupported()) {
...
                             </Link>
-                            <DangerButton v-else-if="lesson.pivot.participation == 'signed_in'"
+                            <DangerButton className="mr-2" v-else-if="lesson.pivot.participation == 'signed_in'"
```
Comment (Marco-Peter) [inline resources/js/Pages/Dashboard.vue:223]:
```suggestion
                            <DangerButton class="mr-2" v-else-if="lesson.pivot.participation == 'signed_in'"
```

File: resources/js/Pages/Dashboard.vue [hunk 3]
```diff
@@ -218,15 +218,15 @@ if (isPushNotificationSupported()) {
...
                                 @click="signOut(lesson)">
                                 Sign Out</DangerButton>
-                            <SecondaryButton v-else-if="lesson.pivot.participation == 'signed_out'"
+                            <SecondaryButton className="mr-2" v-else-if="lesson.pivot.participation == 'signed_out'"
```
Comment (Marco-Peter) [inline resources/js/Pages/Dashboard.vue:226]:
```suggestion
                            <SecondaryButton class="mr-2" v-else-if="lesson.pivot.participation == 'signed_out'"
```

File: resources/js/Pages/Dashboard.vue [hunk 4]
```diff
@@ -218,15 +218,15 @@ if (isPushNotificationSupported()) {
...
                                 @click="signIn(lesson)">
                                 Sign In</SecondaryButton>
-                            <SecondaryButton @click="sendMessage(lesson)">Send Message</SecondaryButton>
+                            <SecondaryButton className="mr-2" @click="sendMessage(lesson)">Send Message</SecondaryButton>
```
Comment (Marco-Peter) [inline resources/js/Pages/Dashboard.vue:229]:
```suggestion
                            <SecondaryButton class="mr-2" @click="sendMessage(lesson)">Send Message</SecondaryButton>
```

**Review by Marco-Peter — CHANGES_REQUESTED** [review]
Please rename `className` to `class`

**Comment by mltucker** [comment]
Ah, the react runs deep in me. Fixed @Marco-Peter

**Pull request merged**